storage/range_test.go

```diff
@@ -835,113 +816,6 @@ func TestRangeIdempotence(t *testing.T) {
 	}
 }
 
-// TestRangeSnapshot.
-func TestRangeSnapshot(t *testing.T) {
```

Comment:
I would think you'd want some tests that the various snapshot methods do what you expect. Am I just missing where this testing is being done?